release-tools/check-crate-version/toml-diff/src/lib.rs

```diff
@@ -0,0 +1,152 @@
+use std::cmp::Ordering;
+use std::mem::discriminant;
+
+use toml::Value as TomlValue;
+
+mod display;
+#[cfg(test)]
+mod test;
+
+pub struct TomlDiff<'a> {
+    pub changes: Vec<TomlChange<'a>>,
+}
+
+#[derive(Debug)]
+pub enum TomlChange<'a> {
+    Added(Vec<&'a str>, &'a TomlValue),
+    Deleted(Vec<&'a str>, &'a TomlValue),
+}
+
+impl<'a> TomlDiff<'a> {
+    /// Return a list of differences between `a` and `b`. A is considered "new" and `b` is
+    /// considered "old", so items missing from `a` are considdered "deletions", while items
```

Comment:
Small nitpick, a few typos of the same word this this doc comment.

`considdered` -> `considered`

Reply:
I don't think I copied those lines, I think my fingers think it's spelled like that lol

Reply:
Fixed